Esterni/AnalisiDeiRequisiti/components/requisiti_qualita.tex

```diff
@@ -0,0 +1,26 @@
+\subsection{Requisiti di qualità}
+{
+    \setlength{\freewidth}{\dimexpr\textwidth-8\tabcolsep}
+    \renewcommand{\arraystretch}{1.5}
+    \centering
+    \setlength{\aboverulesep}{0pt}
+    \setlength{\belowrulesep}{0pt}
+    \rowcolors{2}{AzzurroGruppo!10}{white}
+    \begin{longtable}{C{.20\freewidth} C{.40\freewidth} C{.20\freewidth} C{.20\freewidth}}
+        \toprule
+        \rowcolor{AzzurroGruppo!30}
+        \textbf{Requisito} & \textbf{Descrizione} & \textbf{Classificazione} & \textbf{Fonti} \\
+        \toprule
+        \endhead
+
+        RQO01    & Il prodotto deve essere sviluppato in modo concorde a quanto stabilito nelle \NdP{} \versNdP{} & Obbligatorio & Decisione interna \\
+        RQO02    & Il manuale manutentore deve essere disponibile in lingua italiana & Obbligatorio & Decisione interna \\
```

Comment:
Cambiare
"... Il manuale manutentore ..."
in
"... Il manuale del manutentore ..."